packages/machine/src/middleware/cf-operation/cf-op-setup.ts

```diff
@@ -18,11 +17,14 @@ import {
 
 const { keccak256 } = ethers.utils;
 
-export class CfOpSetup extends CfMultiSendOp {
+export class CfOpSetup extends MultisigTxOp {
   /**
    * Helper method to get hash of an input calldata
+   * @param networkContext
    * @param multisig
-   * @param multisigInput
+   * @param freeBalanceStateChannel
+   * @param freeBalance
+   * @param dependencyNonce
```

Comment:
1. Why is there a docstring here but nowhere else in this source code
2. If there is a docstring it should contain descriptions next to the parameters